anyway/flask_app.py

```diff
@@ -1636,11 +1636,31 @@ def load_user(id: str) -> UserOAuth:
     return db.session.query(UserOAuth).get(id)
 
 
-@app.route("/user/have_email")
+@app.route("/user/info")
 def user_have_email() -> Response:
     if current_user.is_anonymous:
         return return_json_error(Es.BR_USER_NOT_LOGGED_IN)
-    return jsonify({"have_email": bool(get_current_user_email())})
+
+    user_obj = get_current_user()
```

Comment:
check if null before accessing fields

Reply:
In the lines above i am checking that the user is logged in - `if current_user.is_anonymous:` and existing if not,
so this function should never return `None`.

Do you think that the best practice in this case will be to double check it?

Reply:
just went through the documentation, you're right :), my bad.